src/Congo.WebApi/Data/ProductAccess/InsertProductHandler.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Congo.WebApi.Data.Models;
+using MediatR;
+
+namespace Congo.WebApi.Data.ProductAccess
+{
+    public class InsertProductHandler : IRequestHandler<InsertProductCommand>
```

Comment:
This works for now, but it is likely we would want to return the products ID after saving it for future use.